### ISSUE TITLE
Regressions after #5244 

### ISSUE THREAD

**Issue by isshininu** [RPCS3/rpcs3#5271]
After #5244 Dark Souls: Prepare to die edition [BLES01765] went from ingame to loadable and crash with error

F {PPU[0x100000d] Thread (MOMainThread) [0x01c051a4]} MEM: Access violation reading location 0x1c

[RPCS3.7z.gz](https://github.com/RPCS3/rpcs3/files/2497999/RPCS3.7z.gz)

![1412](https://user-images.githubusercontent.com/6324523/47254061-6c66cb80-d46d-11e8-93b1-eaf87813b7b5.png)
p.s. I have been advised to open separate issue.

**Comment by Megamouse** [comment]
you should probably rename it to: "Regressions after #5244", because I bet there will be more

**Comment by isshininu** [comment]
As you wish 😁

**Comment by isshininu** [comment]
hmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmm

**Comment by martinsstuff** [comment]
Another one: Railfan (the Chicago version, BLJM60013) is broken as well after #5244 (crash on white screen). Closing the window crashes RPCS3 with it.

![image](https://user-images.githubusercontent.com/29005772/47261490-76ed8900-d49e-11e8-9600-7f024b577313.png)

(Just an unrelated FYI I wanted to put out there) I also just noticed that some update since I last played it made it go out of control when frames aren't limited such as crashing on the "press play" screen and selections moving to fast to select. On the other hand sound is way better IMO.

**Comment by AniLeo** [comment]
cellDmux is LLE loaded by default, not an issue. 

You're likely forcing manual module loading, which in that case you should know what you're doing, otherwise always use liblv2/Automatic modes.

**Comment by AniLeo** [comment]
Closing as the only reported issue is reported fixed on current master, likely by #5274 

Will be reopened if someone finds other regressions from #5244

**Comment by martinsstuff** [comment]
> cellDmux is LLE loaded by default, not an issue.
> 
> You're likely forcing manual module loading, which in that case you should know what you're doing, otherwise always use liblv2/Automatic modes.

liblv2 or automatic doesn't fix it, while it actually makes it play the intro, game crashes with

```
F {PPU[0x1000087] Thread (HLE Video Decoder)} class std::runtime_error thrown: AU decoding error(0xbebbb1b7): Invalid data found when processing input
(in file c:\projects\rpcs3\rpcs3\emu\cell\modules\cellvdec.cpp:249)
```

When entering the actual game/leaving the menu.

I got the firmware libraries from here: https://forums.rpcs3.net/thread-164106-page-3.html

![image](https://user-images.githubusercontent.com/29005772/47386470-b7d1e180-d6da-11e8-8aef-c38bcbc4e0cd.png)

Again, this was not an issue before, the game ran fine with the previously mentioned manual LLE.

**Comment by AniLeo** [comment]
It was not an issue before because cellDmux HLE was available, now it isn't anymore, so you're forced to LLE load it as well, it's libdmux or something similar.

That error you're getting means broken libvdec HLE implementation, you can use Auto + Manual and only force the video libraries to be manually loaded.

**Comment by martinsstuff** [comment]
Thank you, I managed to get it to work:

New setup is: auto+manual and only manual loaded library being libvdec.

Seems to work except the intro video is kinda stuck but you just have to press the "play" button and it gets skipped.
